docs/usage/configuration-options.md

```diff
@@ -1981,6 +1982,9 @@ If instead you'd prefer to be updated to `^1.2.0` in cases like this, then confi
 
 This feature supports simple caret (`^`) and tilde (`~`) ranges only, like `^1.0.0` and `~1.0.0`.
 
+We discourage using of the `in-range-only` strategy unless you strictly need it.
```

Comment:
How would a user "know" if they really need the `in-range-only` strategy?

The current docs don't really explain if/when/why you would need this strategy.

Reply:
Hi @HonkingGoose I think I tried to explain how this strategy works in this file: docs/usage/configuration-options.md:

- `in-range-only` = Update the lock file when in-range updates are available, ignore package file updates.

Reply:
That is good, but it still doesn't explain why a user of Renovate would want to use the `in-range-only` option.

We as Renovate maintainers/programmers need to explain to our users when they "strictly need" this option. 😉 I think it's too easy on our part to just say: "figure out yourself if you need this or not". 😄

Reply:
Thanks @HonkingGoose I've tried to expand the explanation of the usage of the `in-range-strategy` in docs/usage/configuration-options.md